src/python/pants/option/alias.py

```diff
@@ -47,10 +47,12 @@ class CliOptions(Subsystem):
                 [cli.alias]
                 green = "fmt lint check"
                 all-changed = "--changed-since=HEAD --changed-dependents=transitive"
+                --troubleshoot = "--test-debug --test-force --test-timeout-maximum=300"
```

Comment:
The `all-changed` alias is a option style alias already.

```suggestion
                --all-changed = "--changed-since=HEAD --changed-dependents=transitive"
```